python/lsst/pipe/tasks/processCcd.py

```diff
@@ -63,11 +64,13 @@ class ProcessCcdConfig(pexConfig.Config):
     )
 
     def setDefaults(self):
-        self.isr.doWrite = False
```

Comment:
This does not need to be moved anywhere just set this back to the way it was on master